src/backend/src/yaksuri_progress.c

```diff
@@ -139,7 +139,7 @@ static int alloc_subop(progress_subop_s ** subop)
     yaksuri_request_s *request_backend = (yaksuri_request_s *) elem->request->backend.priv;
     yaksuri_gpudev_id_e id = request_backend->gpudev_id;
     bool need_device_tmpbuf = false, need_host_tmpbuf = false;
-    int devid;
+    int devid = 0;
```

Comment:
Can you initialize this to `INT_MIN` instead, so the code will bomb out if we accidentally use it?